src/renderer/ipc.ts

```diff
@@ -67,3 +67,16 @@ window.electron.on('initiate-undo', async () => {
 window.electron.on('initiate-redo', async () => {
   dispatchRedo();
 });
+
+/**
+ * Used by backend to initiate export from front end
+ */
+window.electron.on('initiate-export-project', async () => {
+  // Retrieve current project state from redux
+  const { currentProject } = store.getState();
+
+  // Don't export if we don't have a project open
+  if (currentProject === null) return;
+
+  await window.electron.exportProject(currentProject);
```

Comment:
Need to update the export file path on the front end as well, you can do this by returning the path used from the back end call

Reply:
Have talked through this with @manikag-12

Reply:
Okay cool. Basically end result is the redux store should be updated to include the export file path that the file was exported to.